pkg/symbolizer/symbolizer.go

```diff
@@ -80,7 +83,11 @@ func (k exeKey) String() string {
 	return fmt.Sprintf("ino=%d mtime=%d.%d", k.ino, k.mtimeSec, k.mtimeNsec)
 }
 
-func NewSymbolizer() (*Symbolizer, error) {
+type SymbolizerOptions struct {
+	GoReSymSpec string
```

Comment:
Please document it. 
- Link to GoReSym
- Specify the format expected. json? is it only the path or the full thing?